### DISCUSSION TITLE
Update xournal

### DISCUSSION THREAD

**Discussion by ghost** [xournalpp/xournalpp#4404] [Q&A] [answered]
hi. i´ve seen it's been released a new version of xournal bot i don't know how to update the app

**Comment by rolandlo** [comment, marked as answer]
Please take a look at the [installation page](https://xournalpp.github.io/installation/)

**Reply by ghost** [reply]
ok, so i just have to download the new version as if i was going to install it for the first time?

**Reply by rolandlo** [reply]
> ok, so i just have to download the new version as if i was going to install it for the first time?

I don't know what OS you use, but basically yes, that's the way to do it.

**Reply by ghost** [reply]
oh, thanks
